libs/utils/src/votecounting.ts

```diff
@@ -339,7 +339,13 @@ function makePairs<T>(inputArray: T[]): Array<Pair<T>> {
   const pairs = [];
   for (let i = 0; i < inputArray.length; i += 1) {
     for (let j = i + 1; j < inputArray.length; j += 1) {
-      pairs.push({ first: inputArray[i], second: inputArray[j] });
+      const first = inputArray[i];
+      const second = inputArray[j];
+      if (!first || !second) {
```

Comment:
this was failing compilation after moving the file, apparently because the build config varies slightly between frontends/election-manager and libs/utils

Reply:
It's the `noUncheckedIndexedAccess` enabled in the shared config that's disabled in `frontends/election-manager`.